src/frontend/Sider/Oppgavebenk/filter/Oppgavefiltrering.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { ChangeEvent, useEffect, useState } from 'react';
```

Comment:
Ikke sett på selve filen, men mener all filtrering burde gjøres i en egen PR etter at det er gjort litt research på hva behovet til SB faktisk er for en oppgavebenk